index.js

```diff
@@ -11,7 +11,11 @@ const handler = async (req, res, client, getCurrentRound) => {
   } else if (segs[0] === 'retrievals' && req.method === 'PATCH') {
     await setRetrievalResult(req, res, client, Number(segs[1]), getCurrentRound)
```

Comment:
Can we remove this route completely? SPARK clients have to update anyway and aren't rewarded yet, so I think we can prioritize the code cleanup now already

Reply:
Got it, ok!